testing/trino-tests/src/test/java/io/trino/tests/ci/TestCiWorkflow.java

```diff
@@ -101,7 +103,7 @@ public void testUploadTestResultsCondition()
     public void testBuildSuccessDependencies()
             throws Exception
     {
-        String buildSuccessJobName = "build-success";
+        String buildSuccessJobName = BUILD_SUCCESS;
```

Comment:
Inline these variables. They don't serve a purpose anymore and are longer than the constant.